ios/playground/Podfile

```diff
@@ -8,6 +8,8 @@ def common
     pod 'SDWebImage', '3.7.5'
     pod 'SocketRocket', '0.4.2'
     pod 'ATSDK-Weex', '0.0.1'
+    
+    pod 'WeexGcanvas', '0.0.1.5'
```

Comment:
I add as follow,is it ok?
# WeexGcanvas is added by Weex Plugin, more info at https://market.dotwe.org/ext/list.htm